iothub/service/samples/PnpServiceSamples/noComponentSample/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Shared;
+
+namespace PnpServiceSample
+{
+    public class Program
+    {        
+        // Get connection string and device id inputs
+        private static readonly string hubConnectionString = Environment.GetEnvironmentVariable("IOTHUB_CONNECTION_STRING");
+        private static readonly string deviceId = Environment.GetEnvironmentVariable("DEVICE_ID");
+
+        // These are default values as per PnP no Component device client sample.
```

Comment:
I wouldn't call these default values, there are property names defined under the DTMI that the device references here. 
You should explicitly call out the DTMI here, and after device samples have been checked in, you can add a link to that.

Reply:
how about this? These are values as defined in DTMI used for PnP no Component device client sample

Reply:
There is a mix of both. I think it is a good idea to add a link to the DTMI but there are a few default values too.